tests/models/vjepa2/test_modeling_vjepa2.py

```diff
@@ -343,3 +343,22 @@ def test_predictor_partial_mask(self):
         # verify the last hidden states
         expected_shape = torch.Size((1, num_masks, 1024))
         self.assertEqual(outputs.predictor_output.last_hidden_state.shape, expected_shape)
+
+    @slow
+    def test_video_classification(self):
+        checkpoint = "qubvel-hf/vjepa2-vitl-fpc16-256-ssv2"
```

Comment:
```suggestion
        checkpoint = "facebook/vjepa2-vitl-fpc16-256-ssv2"
```

prepare for checkpoint's transfer